x-pack/examples/embedded_lens_example/kibana.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "embeddedLensExample",
+  "version": "0.0.1",
+  "kibanaVersion": "kibana",
+  "configPath": ["embedded_lens_example"],
+  "server": false,
+  "ui": true,
+  "requiredPlugins": [
+    "lens",
+    "data",
+    "developerExamples"
```

Comment:
Hey @flash1293! For this example plugin, can we add a dependency on `features` to illustrate that you'll need to [grant access to the `lens` saved object type](https://github.com/elastic/kibana/blob/d4b2a5145a9aa813146e2c587f7be2929f09fb7e/x-pack%2Fplugins%2Ffeatures%2Fserver%2Foss_features.ts#L96) in order to properly embed when security is enabled?

I'm happy to help draft this change if you'd like help with it

Reply:
@legrego Happy to include this - but I'm not sure how it should look like.

This example is not working with saved objects directly, but it does link into the Lens editor. So I guess the right integration would be to check whether there there are at least visualize read permissions via ui capabilities, right?

Reply:
> This example is not working with saved objects directly, but it does link into the Lens editor. So I guess the right integration would be to check whether there there are at least visualize read permissions via ui capabilities and otherwise disable the button, right?

Ah, so this doesn't allow you to view existing visualizations, or save new ones? If that's the case, then you shouldn't need any capabilities checks

Reply:
As discussed offline, I will introduce a third method on the contract which can be used by consumers to determine whether it's possible to link to Lens or not.